docs/snippets/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "docs",
   "version": "2.0.3",
   "description": "A collection code snippets for the Powertools for AWS Lambda (TypeScript) docs",
+  "type": "module",
   "author": {
     "name": "Amazon Web Services",
     "url": "https://aws.amazon.com"
   },
-  "private": true,
```

Comment:
Why removing this?

Reply:
woops, unintentional, the merge was a wild ride.

Reply:
If we remove it, this package will get published to npm and make the release fail.